### ISSUE TITLE
convert_heic_to_jpeg

### ISSUE THREAD

**Issue by Galiwyx** [boredazfcuk/docker-icloudpd#28]
Hello,
What I have to set in Value for the **convert_heic_to_jpeg** variable environnent ?

Second question, what if I set folder structure to **{:%Y/%m}** and 2 pictures have the same name ?

Thank you

**Comment by boredazfcuk** [comment]
> What I have to set in Value for the **convert_heic_to_jpeg** variable environnent ?

It can be anything you like. I just check whether the variable has a value or not. I set to `True` in my containers.

> Second question, what if I set folder structure to **{:%Y/%m}** and 2 pictures have the same name ?

I don't know. The underlying application (https://github.comndbroadbent/icloud_photos_downloader) would be responsible for handling that... I've just saved three jpegs I found on the Internet and it only detects that as a single file. Whether it downloads the first, second or third I couldn't say either. Would need testing.

Fortunately, I've not been in the situation where it would occur for the photos taken by my phone. I set my folder structure to **{:%Y}** but don't take enough pictures in a year to hit the limit... My iPhone sequentially numbers photos from IMG_0000 through to IMG_9999 so I'd have to take 10k pictures in a year before I'd see the issue. With the way you have your folder structure set, you'd have to take 10k pictures in a month.

I'll probably change my folder structure to **{:%Y/%m}** now you've mentioned it though, just to be on the safe side...
